crates/rnote-ui/data/ui/dialogs/import.ui

```diff
@@ -269,4 +269,26 @@
     <property name="lower">1</property>
     <property name="value">96</property>
   </object>
+  <object class="AdwAlertDialog" id="dialog_import_pdf_password">
+    <property name="body" translatable="yes">is password protected</property>
+    <property name="default-response">unlock</property>
+    <property name="close-response">cancel</property>
+    <property name="heading" translatable="yes">Encrypted PDF</property>
+    <property name="follows-content-size">False</property>
+    <responses>
+      <response id="cancel" translatable="yes">_Cancel</response>
+      <response id="unlock" translatable="yes" appearance="suggested">_Unlock</response>
+    </responses>
+    <property name="extra-child">
+    <object class="GtkBox">
```

Comment:
Change this to the following to fix the styling of the password entry. (see: https://gnome.pages.gitlab.gnome.org/libadwaita/doc/1.6/boxed-lists.html)

```suggestion
    <object class="GtkListBox">
      <property name="selection-mode">none</property>
      <style>
        <class name="boxed-list"/>
      </style>
```

Reply:
Resolved in [e87107d](https://github.com/flxzt/rnote/pull/1279/commits/e87107d78fea840eb97b6e1aad11fce62c512b61) 
I opted for animating the ListBox, instead of the entry itself because doing so causes some really funky behaviour. It seems ListBox does not expect its children to be animated.